contrib/clojure-package/project.clj

```diff
@@ -29,7 +29,7 @@
                  ;[org.apache.mxnet/mxnet-full_2.11-linux-x86_64-gpu "1.2.1"]
 
                  ;;; CI
-                 [org.apache.mxnet/mxnet-full_2.11-linux-x86_64-cpu "1.5.0-SNAPSHOT"]
+                 [org.apache.mxnet/mxnet-full_2.11 "1.5.0-SNAPSHOT"]
```

Comment:
Why do you want to remove the platform type from here. ?

Reply:
We removed platform and cpu/gpu specific pom.xml file, the local installed artifact name is now mxnet-full_2.11, clojure won't be able to find mxnet-full_2.11-linux-x86_64-cpu in .m2 any more.